### ISSUE TITLE
Is there an inaccuracy in how the top indicators are determined?

### ISSUE THREAD

**Issue by MarkKoz** [Zer0897/code-jam-5#2]
The top indicators are determined by doing linear regressions on the indicators and comparing their slopes. But the indicators use different data in different units. In one unit a change of 0.1 could be significant but that wont compare well to something that has a change of 1 (slope of 0.1 vs slope of 1).

Also, here are plots of `heat_wave_incidents` and `total_precipitation` respectively:

![bild](https://user-images.githubusercontent.com/1515135/60536569-5e836e80-9cbb-11e9-9afe-0850509edf1b.png)

![bild](https://user-images.githubusercontent.com/1515135/60536574-6216f580-9cbb-11e9-94a1-a7ffc03f24a2.png)

They show an ~8.9% increase and ~0.39% decrease respectively.

**Comment by intendednull** [comment]
Good point. It's hard to know how drastic the effects of those changes are regardless.

It may be better to just get the same indicators for every location.

**Comment by tim-tepia** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @farleyknight, @N1GHTMAR3WALK1N, @poplarsunshine, @imerla1, @progsmile, @crescenity, @0xJuancito, @silemelon, @dmurchison, @Notmeor, @lakamson, @andeya, @zeroonez, @machine-creek, @fooock, @Ksj14-kumar, @GJen484, @silasms, @julianclatro, @strixaluco, @kensecretum, @omerkarabacak, @JesseAldridge, @exdx, @itsnotvalid, @AbhishekGit-hash, @martingaido, @inglkruiz, @Liukemeng, @biezhihua, @adityavs, @metanet4jDev, @x7c7v7i87, @hailelagi, @xialvjun, @foshougua, @symnoisy, @AkhyarRamadan, @rosman83, @chuhemiao, @georgelima, @jinuhwang, @b4s36t4, @karthikspedx, @thurgan20, @mrsaifullah52, @akxcix, @daji777, @zhangzhu16, @bebrws
